core-bundle/tests/Translation/DataCollectorTranslatorTest.php

```diff
@@ -98,7 +97,6 @@ public function testServiceIsResetable(): void
 
         $translator = new DataCollectorTranslator($originalTranslator);
 
-        $this->assertInstanceOf(ResetInterface::class, $translator);
```

Comment:
Why this change?

Reply:
See https://github.com/contao/contao/pull/7711#discussion_r1837898458